dat/missions/dvaered/colonel-transport.lua

```diff
@@ -0,0 +1,164 @@
+                This is a one-off that's not part of any major
+                storyline; the in-game purpose will remain a mystery.
+
+--]]
+
+local escort = require "escort"
+local fmt = require "format"
+local lmisn = require "lmisn"
+local neu = require "neutral"
+local vn = require "vn"
+local vni = require "vnimage"
+
+local reward = 750e3
+
+local source_system = nil
+
+local npc_name = _("Dvaered Colonel")
+local npc_portrait = nil
+local npc_image = nil
+
+local m = vn.newCharacter( npc_name, {image=npc_image} )
+
+function create()
+   local spbs = lmisn.getSpobAtDistance( nil, 2, 6, "Dvaered" )
+   if #spbs <= 0 then misn.finish(false) end
+   mem.destspb = spbs[ rnd.rnd(1,#spbs) ]
+   mem.dest_sys = mem.destspb:system()
+   mem.npc_image, mem.npc_portrait = vni.dvaeredMilitary()
+   misn.setNPC(npc_name, npc_portrait, _("A Dvaered, very professional-looking, is sitting with an excellant posture at the bar.") )
+end
+
+local function factions()
+   local dvaered_factions = faction.get("Dvaered")
+
+   local ffriendly = faction.dynAdd( dvaered_factions, "radver", _("Dvaered") ) -- codespell:ignore ffriendly
+   local fhostile = faction.dynAdd( dvaered_factions, "radver_baddie", _("Dvaered Warlord") )
+   faction.dynEnemy( ffriendly, fhostile ) -- codespell:ignore ffriendly
+   return ffriendly, fhostile -- codespell:ignore ffriendly
+end
+
+function accept()
+   local accepted = false
+   vn.clear()
+   vn.scene()
+   vn.transition()
+   m(fmt.f([[As you approach, the Dvaered soldier stands. "Oh, is it Captain {playername}?" they say. "Nice to see you around here. How's it going?"]]))
+      {playername=player.name()}
+   vn.menu{
+      {_([["Quite well, thank you!"]]), "well"},
+      {_([["I guess it's going alright."]]), "fine"},
+      {_([["Wait a minute. How did you know my name?"]]), "my name"},
+   }
+
+   vn.label("well")
+   m(_([["Alright."]]))
+   vn.jump("mission description")
+
+   vn.label("fine")
+   m(_([["As long as there's no trouble…"]]))
+   vn.jump("mission description")
+
+   vn.label("my name")
+   m(_([["Uhh… tough luck, captain, I'm not allowed to tell you. Too bad.]]))
+   m(_([["Let's just say that my espionage services are of the best, and leave it at that… shall we?"]])) -- When/if possible, the word "shall" should be italicized.
+   vn.jump("mission description")
+
+   vn.label("mission description")
+   m(fmt.f([["Well, to the point. I need somebody who's not crazy to escort me and my Arsenal to {pnt} in the {sys} system. Would you do that? I can't tell you why, that would be classified information leaked to an outsider. Very dangerous, especially when we can't trust you.]]))
+     {pnt=mem.destspb, sys=mem.dest_sys}
+   m(_([["One more thing: another warlord would be only to happy to blow any of the colonel rank such as myself up, so you may need to expect attacks. The ship will probably be called 'Asheron Anomaly', but it may be something else."]]))
```

Comment:
You should probably add a menu option to ask about the `Asheron Anomaly`, because I'm pretty sure it'll stand out to players.